e2e/test/scenarios/onboarding/setup/setup.cy.spec.ts

```diff
@@ -9,9 +9,9 @@ const { admin } = USERS;
 const locales = ["en", "xx"];
 
 describe("scenarios > setup", () => {
-  locales.forEach((locale) => {
-    beforeEach(() => H.restore("blank"));
+  beforeEach(() => H.restore("blank"));
 
+  locales.forEach((locale) => {
```

Comment:
`beforeEach` was inside this loop! I think this was an oversight on my part 🤦 
We were restoring the "blank" snapshot two times per test.

Reply:
wow... you have better eyes than me. I totally didn't notice this 🤦